donjo-app/views/nav.php

```diff
@@ -2,11 +2,32 @@
 <!-- Nantinya seluruh file nav.php (sub modul) yang ada di masing-masing folder modul utama akan dihapus (sudah tidak digunakan lagi)  -->
 <aside class="main-sidebar">		
 	<section class="sidebar">	
+		<div class="user-panel">
+			<div class="pull-left image">
+				<img src="<?php echo LogoDesa($desa['logo']);?>" class="img-circle" alt="User Image">
+			</div>        
+			<div class="pull-left info">
+				<p><?php echo ucwords($this->setting->sebutan_desa." ".$desa['nama_desa']); ?></p>
+				<?php 
+					$nam_kec = strlen(unpenetration($desa['nama_kecamatan']));
+					$nam_kab = strlen(unpenetration($desa['nama_kabupaten']));
+					if ($nam_kec<=12 AND $nam_kab<=12){
+						echo ucwords($this->setting->sebutan_kecamatan." ".unpenetration($desa['nama_kecamatan']));
+						echo "</br>";
+						echo ucwords($this->setting->sebutan_kabupaten." ".unpenetration($desa['nama_kabupaten']));
+					}else{
+						echo ucwords(substr($this->setting->sebutan_kecamatan,0,3).". ".unpenetration($desa['nama_kecamatan']));
+						echo "</br>";
+						echo ucwords(substr($this->setting->sebutan_kabupaten,0,3).". ".unpenetration($desa['nama_kabupaten']));
+					}
+					
+				?>				
+				
+			</div>
+		</div>	
 		<ul class="sidebar-menu" data-widget="tree">
 			<li class="header">MENU UTAMA</li>
```

Comment:
~~~
<?php foreach ($modul as $mod):?>
	<?php
		$this->modul_ini == $mod['id'] AND $class = 'active';
		count($mod['submodul'])>0 AND $class .= 'treeview';
		$mod['icon'] OR $mod['icon'] = 'fa-circle-o';
	?>
	<li class="<?= $class ?>">
		<i class="<?= $mod['icon'] ?>"></i>
		...
	</li>
<?php endforeach ?>
~~~